Assets/Editor/StandaloneBuild/Builder.cs

```diff
@@ -56,10 +56,14 @@ public static class AutoBuilder {
 						// all scenes specified in build config must be in the Scenes folder
 						string scenePath = Application.dataPath + "/Scenes/" + s.Path;
 						if (File.Exists(scenePath)) {
-							// found a file
-							// write the name of the scene to ScenesList
-							// no other path information, no file extension
-							string sceneName = scenePath.Remove(0, Application.dataPath.Length - "Assets".Length);
+                            // found a file
+                            // write the name of the scene to ScenesList
+                            // no other path information, no file extension
+                            // jin: sceneName seems to be redundant and probably could be deleted according to the comments above.
```

Comment:
name will show up in `git blame`, no need to explicitly put names in the code lines.